### DISCUSSION TITLE
Where are v2 docs?

### DISCUSSION THREAD

**Discussion by brandonjp** [alpinejs/alpine#1490] [1. Help] [answered]
Where are v2 docs?

**Comment by jeffchown** [comment, marked as answer]
Have a look here: https://github.com/alpinejs/alpine/tree/v2.8.2

**Reply by brandonjp** [reply]
Thanks @jeffchown  //  @fschweikhardt see here 👆

**Reply by calebporzio** [reply]
@brandonjp - good call - I'll add a link to the README
